realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionChange.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Date;
 import java.util.UUID;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
Does that mean as long as we don't have `SuppressFBWarnings` in the proxy classes, user won't see the warning?